### PR TITLE
Updates to Windows installation procedure

### PR DESCRIPTION
**Pull request by friism**

### CODE REVIEW

**Comment by friism** [comment]
@gbarr01 where in the text is `https://download.docker.com/win/stable/InstallDocker.exe` mentioned?

File: docker-for-windows/install.md [hunk 1]
```diff
@@ -122,13 +122,18 @@ Windows containers.
 
 ## Install Docker for Windows
 
-1. Double-click `InstallDocker.msi` to run the installer.
+1. Double-click **Docker for Windows Installer.exe** to run the installer.
 
-    If you haven't already downloaded the installer (`InstallDocker.msi`), you can get it [**here**](https://download.docker.com/win/stable/InstallDocker.msi). It typically downloads to your `Downloads folder`, or you can run it from the recent downloads bar at the bottom of your web browser.
+    If you haven't already downloaded the installer (`Docker for Windows Installer.exe`), you can get it from
+    [**download.docker.com**](https://download.docker.com/win/stable/Docker%20for%20Windows%20Installer.exe).
```
Comment (mstanleyjones) [inline docker-for-windows/install.md]: This link doesn't really exist. At least not for me.

**Comment by gbarr01** [comment]
Hi, friism, it's the URI to the link "here" that msj changed to download.docker.com: 

```
[**here**](https://download.docker.com/win/stable/Docker%20for%20Windows%20Installer.exe)
[**download.docker.com**](https://download.docker.com/win/stable/Docker%20for%20Windows%20Installer.exe)
```

It looks like you changed the URI by changing the extension from .msi to .exe. But there is no such exe at that address.

**Comment by friism** [comment]
https://youtu.be/2MHGovG5MWw

**Pull request merged**